lib/harmony/harmonydb/sql/20230919.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE harmony_config (
+    id SERIAL PRIMARY KEY NOT NULL,
+    title VARCHAR(300) UNIQUE NOT NULL,
```

Comment:
Probably being nitpicky here:
* Why use varchar instead of simple text?
* Why have unique title and id? Can we just have the title?
* Do we want a separate `type` so that we can support other node types can also use this?
* super-nit: maybe use `name` instead of `titile`?

Reply:
Titles should have reasonable length limits. 
id is far better for if we ever need to do joins. 
type: A single text blob covering all products makes better sense to me so they don't need to duplicate anything. 
name is a SQL reserved word: it's doable but harder to work with: `name`

Reply:
along with being convention, internally most databases will create this column if you do not. The only way to access it is at creation time.